src/pages/selection/candidates/index.jsx

```diff
@@ -1,76 +1,105 @@
 import React, { Component } from 'react';
 import SectionHeader from 'components/SectionHeader';
 import pyv from 'utils/api/pyv';
-import {IMAGE_BASE} from 'utils/image';
+import { IMAGE_BASE } from 'utils/image';
 
 class Candidates extends Component {
   _isMounted = false;
   state = {
-    candidates: [],
+    races: [],
     candidatesHeader: {
       pageTitle: '',
       pageDescription: ''
-    }
+    },
+    selectedCandidates: []
   };
+
   componentDidMount() {
     this._isMounted = true;
     this.loadCandidatesApi().then(data => {
       if (this._isMounted) {
         this.setState({
-          candidates: data[0], 
+          races: data.races,
           candidatesHeader: {
-            pageTitle: data[1].votingPage.pageTitle,
-            pageDescription: data[1].votingPage.pageDescription,
-            pageNum: data[1].votingPage.pageNumber
+            pageTitle: data.votingPage.pageTitle,
+            pageDescription: data.votingPage.pageDescription,
+            pageNum: data.votingPage.pageNumber
           }
         });
       }
-    }
-    );
+    });
   }
   componentWillUnmount() {
     this._isMounted = false;
   }
   loadCandidatesApi = async () => {
-    const response = await pyv.get('/api/candidates');
-    const response2 = await pyv.get('/api/races'); 
-    return [response.data, response2.data];
+    // const response = await pyv.get('/api/candidates');
+    const response = await pyv.get('/api/races');
+    return response.data;
+  };
+
+  selectBtn = data => {
+    let temp = {};
```

Comment:
Wouldn't it be better to go like this?

```javascript
const temp = {
  candidateId: data.candidateId,
  name: data.name,
  electionId: data.electionId,
  election: data.election,
  details: data.details,
  organizationId: data.organizationId,
  organization: data.organization,
  candidateRaces: data.candidateRaces,
  contacts: data.contacts
};
```

Reply:
because:
1. better readability
2. we're not going to change `temp` so I think `const` would be a better choice